packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -318,9 +318,57 @@ class DateTimePicker extends React.Component {
 
   handleTimeChange(event, time) {
     this.timeValue = time;
-    const validDate = DateUtil.isValidDate(this.dateValue, this.state.dateFormat) && this.isDateTimeAcceptable(DateTimeUtils.convertDateTimeStringToMomentObject(this.dateValue, this.timeValue, this.state.dateFormat, this.props.showSeconds, this.initialTimeZone));
+    let validDate = DateUtil.isValidDate(this.dateValue, this.state.dateFormat) && this.isDateTimeAcceptable(DateTimeUtils.convertDateTimeStringToMomentObject(this.dateValue, this.timeValue, this.state.dateFormat, this.props.showSeconds, this.initialTimeZone));
     const validTime = DateTimeUtils.isValidTime(this.timeValue, this.props.showSeconds);
-    const previousDateTime = this.state.dateTime ? this.state.dateTime.clone() : null;
+    let previousDateTime = this.state.dateTime ? this.state.dateTime.clone() : null;
+    let dateChanged = false;
+
+    if (event.type === 'keydown') {
+      const timeArray = time.split(':');
+      if (event.key === 'n' || event.key === 'N') {
+        previousDateTime = DateUtil.createSafeDate(DateUtil.getCurrentDate(), this.initialTimeZone);
+        validDate = true;
+        dateChanged = true;
+      }
+
+      if (event.key === '-' || event.key === '_') {
+        if (!validDate) {
```

Comment:
There appears to be an issue when calculating the missing hour when daylight saving begins in the Spring. When the time is `3:00` and using the minus sign `-` to subtract the minute it is expected to go to `1:59` but instead it cycles back to `3:59`.
![Feb-24-2022 18-19-40](https://user-images.githubusercontent.com/22618655/155629674-b3dfa346-c65e-4b2d-b02a-1ef8be70675e.gif)

Reply:
This bit of logic should solve it for `-`, without effecting Fall DST hour issues. Feel free to test it https://github.com/cerner/terra-framework/pull/1520/commits/e31d2250dc60b5bc379214c65e7312ff889cf584#diff-063daa4349be2bbad9c58209d280bfaee7a04b7e429163ebdb7784768f501a5cR345-R352

`+` didn't need to be fixed because the auto skipping of the spring hour worked in its favor to just auto bump up.